assets/js/controllers/coinify/coinifyCheckout.controller.js

```diff
@@ -0,0 +1,52 @@
+angular
+  .module('walletApp')
+  .controller('CoinifyCheckoutController', CoinifyCheckoutController);
+
+function CoinifyCheckoutController ($scope, $rootScope, $stateParams, Env, AngularHelper, MyWallet, $state, Wallet, currency, coinify, modals, balance) {
+  $scope.exchange = coinify.exchange;
+
+  $scope.trades = coinify.trades;
+
+  $scope.fiatOptions = currency.coinifyCurrencies;
+  let walletCurrencyMatch = $scope.fiatOptions.filter(c => c.code === ($scope.exchange.profile ? $scope.exchange.profile.defaultCurrency : Wallet.settings.currency.code))[0];
```

Comment:
This wasn't working when I looked at it with a wallet currency set to DKK. My currency was set to EUR every time.